Data/core_rulebook/cr_skills.lua

```diff
@@ -0,0 +1,8771 @@
+-- Converted From LST file data\pathfinder\paizo\roleplaying_game\core_rulebook\cr_skills.lst
+-- From repository https://github.com/pcgen/pcgen at commit 11ceb52482855f2e5f0f6c108c3dc665b12af237
+SetSource({
+  SourceLong="Core Rulebook",
+  SourceShort="CR",
+  SourceWeb="http://paizo.com/store/downloads/pathfinder/pathfinderRPG/v5748btpy88yj",
+  SourceDate="2009-08",
+})
+DefineSkill({
+  Name="Acrobatics",
+  Types={
+    "Dexterity",
+    "ACHECK",
+    "Base",
+  },
+  KeyStat="DEX",
+  ArmorCheckPenalty="Yes",
+  Bonuses={
+    {
+      Category="SKILL",
+      Variables={
+        "Acrobatics",
+      },
+      Formula=Formula("3"),
+      Type={
+        Name="ClassSkill",
+        Replace=false,
+        Stack=false,
+      },
+      Conditions={
+        function (character)
+          return ((character.IsClassSkill("Acrobatics") and 1 or 0)) >= 1
```

Comment:
"and 1 or 0"... the greater than 1 is weird too, since it's a boolean. :-) Is "true" >= 1?  or does "character.IsClassSkill" have to return a number?

Reply:
This is a generic condition thingy that is "n of these things must be true" and sometimes its like 2 things must be "this type"
so each condition is "how many things match the condition"